docs/_build/html/_sources/examples.rst.txt

```diff
@@ -0,0 +1,357 @@
+********
+Examples
+********
+
+
+:py:func:`sunrise.timestamp_to_hour`
+------------------------------------
+
+Example:
+Take the example of six minutes into January in 2017.
+
+>>> import pandas as pd
+>>> timestamp = pd.Timestamp('2017-01-01T0006')
+>>> print(timestamp)
+    2017-01-01 00:06:00
+>>> timestamp_to_hour(timestamp)
+    24.1
+
+
+:py:func:`sunrise.hour_number`
+------------------------------
+
+Example:
+Take the example of six minutes into January first
+
+>>> N = 1
+>>> time = 0.1
+>>> hour_number(N, time)
+    24.1
+
+
+:py:func:`sunrise.day_number`
+-----------------------------
+
+Example:
+Midnight on day 366 of a leap year
+
+>>> hour_number = 8784
+>>> day_number(hour_number)
+    366
+
+
+:py:func:`sunrise.local_time`
+-----------------------------
+
+Example:
+Midnight on the third day of a year.
+
+>>> N_days = 3
+>>> time = 24
+>>> hour_number = hour_number(3, 24)
+>>> local_time(hour_number)
+    0
+
+For a period of two days, the function local_time behaves as
+
+>>> q = np.linspace(0,48,480).tolist()
+>>> ts = []
+>>> for i in q:
+>>>     ts.append(local_time(i))
+>>> plt.plot(q,ts)
+>>> plt.title('Local Time Plot')
+>>> plt.xlabel('Hour Number')
+>>> plt.ylabel('Local Time')
+>>> plt.xticks(np.arange(0, 56, step=8))
+>>> plt.yticks(np.arange(0, 32, step=8))
+
+.. image:: ../examples/plots/lt_2day.png
+
+
+:py:func:`sunrise.frac_year`
+----------------------------
+
+Example:
+Six minutes into the third day of a leap year.
+It is important to remember that for a leap year you
+must put the True Boolean as the second parameter.
+If you want a non-leap year you can either put only
+one argument or use the False Boolean.
+
+>>> hour = hour_number(3, 0.1)
+>>> frac_year(hour, True)
+    -76.7172131147541
+    
+For a period of one year, the function frac_year behaves as
+
+>>> w = np.linspace(0,8760,87600).tolist()
+>>> ts = []
+>>> for i in w:
+>>>     ts.append(frac_year(i))
+>>> plt.plot(w,ts)
+>>> plt.title('Year Fraction')
+>>> plt.xlabel('Hour Number')
+>>> plt.xticks(np.arange(0, 9000, step=895))
+>>> plt.yticks(np.arange(-77, 340, step=100))
+
+.. image:: ../examples/plots/yf_1year.png
+
+
+:py:func:`sunrise.declination`
+------------------------------
+
+Example:
+The declination six minutes into January third of
+a leap year can be calculated as follows. It is
+important to remember that for a leap year you
+must put the True Boolean as the second parameter.
+If you want a non-leap year you can either put only
+one argument or use the False Boolean.
+
+>>> hour = hour_number(3,0.1)
+>>> declination(hour, True)
+    -22.81293175279647
+
+For a period of one year, the function declination behaves as
+
+>>> w = np.linspace(0,8760,87600).tolist()
+>>> ts = []
+>>> for i in w:
+>>>     ts.append(declination(i))
```

Comment:
```suggestion
>>> ts = [declination(i) for i in w]
```
This can be one line with list comprehension in python.

I know there are others, you might consider changing.